backend/app.js

```diff
@@ -1,7 +1,123 @@
 const express = require("express");
 const app = express();
 const port = 3000;
+const cors = require("cors");
+app.use(cors());
 
-app.get("/", (req, res) => res.send("Hello World!"));
+app.get("/orders", function(req, res) {
+  // Authorization
+  fs.readFile("credentials.json", (err, content) => {
+    if (err) return console.log("Error loading client secret file:", err);
+    // Authorize a client with credentials, then call the Google Sheets API.
+    authorize(JSON.parse(content), listMajors);
+  });
+
+  /**
+   * Prints the names and majors of students in a sample spreadsheet:
+   * @see https://docs.google.com/spreadsheets/d/1ZNnltFhQluexcZrwWpuaIlMLtDQTeAI2WsoRHWgmELg/edit#gid=0
+   * @param {google.auth.OAuth2} auth The authenticated Google OAuth client.
+   */
+  function listMajors(auth) {
+    const sheets = google.sheets({ version: "v4", auth });
+    sheets.spreadsheets.values.get(
+      {
+        // (1) Updating function to use batchGet()
+        spreadsheetId: "1ZNnltFhQluexcZrwWpuaIlMLtDQTeAI2WsoRHWgmELg",
+        // (2) Updating "range"
+        range: "A2:G"
+      },
+      (err, response) => {
+        if (err) return console.log("The API returned an error: " + err);
+        // (3) Pulling first two ranges into arrays - data format changes from.values to .valueRanges[range defined above].values
+        const rows = response.data.values;
+        if (rows.length) {
+          console.log(
+            "Pick Up Date, LastName, FirstName, Size/Style, ItemName" +
+              ", Quantity, Picked up?: "
+          );
```

Comment:
It's ok to have console.log statements to debug but remove them before merging in